website/app/styles/components/do-dont.scss

```diff
@@ -26,17 +28,17 @@
 }
 
 .doc-do-dont__content {
-  padding: 12px;
+  padding: 16px;
   border-style: solid;
-  border-width: 1px;
+  border-width: 3px 1px 1px 1px;
   border-radius: 3px;
 
   @include breakpoint.medium () {
-    padding: 18px;
+    padding: 24px;
   }
 
   @include breakpoint.large () {
-    padding: 24px;
+    padding: 32px;
```

Comment:
I'll connect with @cveigt on this, but I'd consider the content within the do/don't is going to be more similar to what is in the [code block](https://www.figma.com/file/Ky0qWjvHZR3je1lCBlzNB1/HDS-website?node-id=1394%3A93953&t=SsdDnvBI8nyMu98k-1) which matches this pattern, rather than the banner (just text-based).

Reply:
We've determined that the banner and the do/don't block don't need the same padding:
* Do/don't benefits from more padding initially, similar to the code block
* Banner benefits from tighter padding based on the content